storage/sql/migrate.go

```diff
@@ -61,14 +63,14 @@ func (c *conn) migrate() (int, error) {
 }
 
 type migration struct {
-	stmt string
+	stmts []string
```

Comment:
Why do we need this? I haven't dug into this code in a while, but I'd think we could wrap stuff in `BEGIN; ... COMMIT;`, maybe? What's wrong with the `;` concatenation? 🤔I guess it's difficult to spot what's wrong; transactions-wise, I think it should not matter, should it?

Reply:
The issue here is that MySQL driver by default doesn't support multiple statements in one execution, in my original implementation I have used https://github.com/go-sql-driver/mysql#multistatements and this change is not needed. If you think that is better I can use that version.